tests/cypress/reports/mochawesome.json

```diff
@@ -0,0 +1,110 @@
+{
```

Comment:
It looks like this report file is accidentally committed to the repo, Could you Please add the `tests/cypress/reports` folder to `.gitignore`? It contains test report files and shouldn't be pushed to the repo.